### ISSUE TITLE
question about pred_bboxes.detach() in DynamicHead.forward()

### ISSUE THREAD

**Issue by burningisnaive** [PeizeSun/SparseR-CNN#81]
In the forward method of DynamicHeadm, pred_bboxes are detached from graph before being used as input of next iteration, but proposal_features are kept. Why not just detach both or keep both at the same time?

**Comment by PeizeSun** [comment]
Hi~
We tried detach both, keep both, detach one and keep another, current version is the setting with best result.

**Comment by burningisnaive** [comment]
Thanks for your reply
